hub/core/storage/readme.md

```diff
@@ -0,0 +1,16 @@
+# Temporary document, will be added to ReadTheDocs later
```

Comment:
Let's not make a habit of committing temporary documents.

Reply:
Shashank asked me to commit this for now, until the readthedocs is up